backend/docker-compose_init.bat

```diff
@@ -0,0 +1,3 @@
+@echo off
+docker-machine start default
+call docker-compose up

```

Comment:
we also need a `--build` key so compose will reflect changes in dockerfile even after first container was created